src/models/AccessLog.ts

```diff
@@ -17,26 +17,26 @@ export enum EntityType {
 @Entity()
 export class AccessLog {
   @PrimaryGeneratedColumn('uuid')
-  id: string;
+    id: string;
 
-  @Column()
-  userId: string;
+    @Column()
+      userId: string;
```

Comment:
It's strange that these fields are having an indentation like that, possibly there is an error in the eslint configuration
Please fix